src/praisonai-agents/praisonaiagents/guardrails/llm_guardrail.py

```
@@ -132,5 +138,117 @@ def __call__(self, task_output) -> Tuple[bool, Union[str, "TaskOutput"]]:
+Please respond with either:
+- "PASS" if the content meets the criteria
+- "FAIL: [reason]" if the content does not meet the criteria
+
+Your response:"""
+
+            # Get LLM response
+            if hasattr(self.llm, 'complete'):
+                response = self.llm.complete(prompt)
+            elif hasattr(self.llm, 'invoke'):
+                response = self.llm.invoke(prompt)
+            elif hasattr(self.llm, '__call__'):
```

Ruff (0.15.9)

[warning] Using `hasattr(x, "__call__")` to test if x is callable is unreliable. Use `callable(x)` for consistent results.

Replace with `callable()`

(B004)